build/common.props

```diff
@@ -1,7 +1,7 @@
 <Project>
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;</TargetFrameworks>
```

Comment:
revert?

Reply:
Nope. We are overriding TargetFrameworks and if you don't have multiple or a ; it gives a warning.